### PR TITLE
Release v1.12.1 RC1

### PR DESCRIPTION
**Pull request by GretaD**
Ref https://github.com/nextcloud/groupware/issues/39

### CODE REVIEW

**Comment by GretaD** [comment]
im not sure if i need the changelog here, but added it just to make sure

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.12.1 - 2022-05-23
```
Comment (ChristophWurst) [inline CHANGELOG.md:4]:
```suggestion
## 1.12.1 - 2022-05-24
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+### Added
+- Consider passwordless signings when provisioning
```
Comment (ChristophWurst) [inline CHANGELOG.md:6]: this is a bugfix. There are no *Added*/features in a patch release

File: appinfo/info.xml [hunk 3]
```diff
@@ -12,7 +12,7 @@
 - **🙈 We’re not reinventing the wheel!** Based on the great [Horde](https://horde.org) libraries.
 - **📬 Want to host your own mail server?** We don’t have to reimplement this as you could set up [Mail-in-a-Box](https://mailinabox.email)!
 	]]></description>
-	<version>1.12.0</version>
+	<version>1.12.1.rc1</version>
```
Comment (ChristophWurst) [inline appinfo/info.xml:15]:
```suggestion
	<version>1.12.1-rc.1</version>
```

File: package-lock.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nextcloud-mail",
-  "version": "1.12.0",
+  "version": "1.12.1.rc1",
```
Comment (ChristophWurst) [inline package-lock.json:3]:
```suggestion
  "version": "1.12.1",
```

File: package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "nextcloud-mail",
   "description": "Nextcloud Mail",
-  "version": "1.12.0",
+  "version": "1.12.1.rc1",
```
Comment (ChristophWurst) [inline package.json:4]:
```suggestion
  "version": "1.12.1",
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: appinfo/info.xml [hunk 6]
```diff
@@ -12,7 +12,7 @@
 - **🙈 We’re not reinventing the wheel!** Based on the great [Horde](https://horde.org) libraries.
 - **📬 Want to host your own mail server?** We don’t have to reimplement this as you could set up [Mail-in-a-Box](https://mailinabox.email)!
 	]]></description>
-	<version>1.12.0</version>
+	<version>1.12.1.rc.1</version>
```
Comment (ChristophWurst) [inline appinfo/info.xml:15]:
```suggestion
	<version>1.12.1-rc.1</version>
```

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**